Example/Emission/UnroutedViewController.m

```diff
@@ -37,7 +37,7 @@ - (NSURL *)urlForRoute:(NSString *)route
 
   BOOL useStaging = [[NSUserDefaults standardUserDefaults] boolForKey:ARUseStagingDefault];
   NSURL *url = [NSURL URLWithString: useStaging ? @"https://staging.artsy.net" : @"https://artsy.net"];
-  return [url URLByAppendingPathComponent:route];
+  return [NSURL URLWithString:route relativeToURL:url];
```

Comment:
Since the new route contains a query string, it isn't strictly a path component. This moves to `NSURL`'s recommended way to make a relative URL absolute.